ui/src/app/components/webcomponents/test-plan-test-machine-form.component.ts

```diff
@@ -34,7 +34,6 @@ export class TestPlanTestMachineFormComponent extends TestPlanPlatformOsVersionF
     }
     if(this.isHybrid){
       this.setTargetMachines();
-      this.fetchPlatForms();
```

Comment:
@Raksha-testsigma Can you explain why we are not fetching platforms if it is hybrid execution ?

Reply:
We use platforms only for Testsigma Labs, and it is unnecessary while performing Local Agent Executions. Hence removed the call, -similar behavior is observed in cloud implementation as well.